types/bunyan-slack/tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": false,
```

This should be `true`. You have an implicit `any` in your definition of `BunyanSlackConstructor` as the definition of `new(...)` does not have a return type, and that would have been caught when testing.
```suggestion
        "noImplicitAny": true,
```